python/emokit/emotiv.py

```diff
@@ -302,9 +306,9 @@ def handle_quality(self, sensors):
         Optionally will return the value.
         """
         if self.old_model:
-            current_contact_quality = get_level(self.raw_data, quality_bits) / 540
+            current_contact_quality = get_level(self.raw_data, quality_bits) / 540.0
         else:
-            current_contact_quality = get_level(self.raw_data, quality_bits) / 1024
+            current_contact_quality = get_level(self.raw_data, quality_bits) / 1024.0
         sensor = ord(self.raw_data[0])
```

Comment:
There is a reason this is an integer as well. Easy identification of signal level in the pygame visual. I don't believe it needs to be a floating point value.